support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummarySelector.js

```diff
@@ -19,19 +21,34 @@ const getPromotion = (productPrice: ProductPrice): Option<?string> =>
     : null);
 
 function mapStateToProps(state: CheckoutState): EndSummaryProps {
-  const { billingPeriod, productPrices } = state.page.checkout;
-
-  const productPrice = getProductPrice(
-    productPrices,
-    state.common.internationalisation.countryId,
-    billingPeriod,
-  );
+  const {
+    billingPeriod, productPrices, fulfilmentOption, productOption, product,
+  } = state.page.checkout;
+
+  const productPrice = (product === 'Paper') ?
+    getPaperProductPrice(
+      productPrices,
+      fulfilmentOption,
+      productOption,
+    ) :
+
+    getProductPrice(
+      productPrices,
+      state.common.internationalisation.countryId,
+      billingPeriod,
+    );
 
   const digitalBillingPeriod = billingPeriod === 'Annual' ? billingPeriod : 'Monthly';
 
+  const livePaperSubscriptionOption = productOption === 'Everyday' || productOption === 'Sixday' || productOption === 'Weekend' || productOption === 'Sunday' ? productOption : null;
```

Comment:
```suggestion
const activePaperOption = ActivePaperProductTypes.find(p => p === productOption);
```

Reply:
`ActivePaperProductTypes` is in productOptions.js

Reply:
Thanks!